libs/platform/src/lib/components/form/multi-combobox/multi-combobox-mobile/multi-combobox/multi-combobox-mobile.component.ts

```diff
@@ -0,0 +1,127 @@
+import {
+    ChangeDetectionStrategy,
+    Component,
+    ElementRef,
+    Inject,
+    OnDestroy,
+    OnInit,
+    Optional,
+    TemplateRef,
+    ViewChild
+} from '@angular/core';
+import { takeUntil } from 'rxjs/operators';
+
+import {
+    DialogService,
+    MOBILE_MODE_CONFIG,
+    MobileModeBase,
+    MobileModeConfigToken,
+    MobileModeControl
+} from '@fundamental-ngx/core';
+
+import { MULTICOMBOBOX_COMPONENT, MultiComboboxInterface } from '../../multi-combobox.interface';
+import { SelectableOptionItem } from '../../../../../domain';
+
+@Component({
+    selector: 'fdp-multi-combobox-mobile',
+    templateUrl: './multi-combobox-mobile.component.html',
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class MultiComboboxMobileComponent extends MobileModeBase<MultiComboboxInterface> implements OnInit, OnDestroy {
+    /** @hidden */
+    @ViewChild('dialogTemplate') dialogTemplate: TemplateRef<any>;
+
+    /** @hidden
+     * For internal usage
+     * Control element, which will be rendered inside dialog.
+     * List element, which will be rendered inside dialog.
+     */
+    childContent: {
+        listTemplate: TemplateRef<any>,
+        controlTemplate: TemplateRef<any>
+    } = null;
+
+    /** @hidden */
+    selectedShown$ = this._component.selectedShown$;
+
+    /** @hidden */
+    private _selectedBackup: SelectableOptionItem[];
+
+    constructor(
+        elementRef: ElementRef,
+        dialogService: DialogService,
+        @Inject(MULTICOMBOBOX_COMPONENT) multiComboboxComponent: MultiComboboxInterface,
+        @Optional() @Inject(MOBILE_MODE_CONFIG) mobileModes: MobileModeConfigToken[]
+    ) {
+        super(elementRef, dialogService, multiComboboxComponent, MobileModeControl.MULTI_COMBOBOX, mobileModes);
+    }
+
+    /** @hidden */
+    ngOnInit(): void {
+        this._listenOnMultiComboboxOpenChange();
+    }
+
+    /** @hidden */
+    ngOnDestroy(): void {
+        super.onDestroy();
```

Comment:
Do we should have it here? As  the parent lifecycle methods will be called by default if no overrides provided.

Reply:
@platon-rov yeah, we should. Unfortunately, `BaseInput.ngOnDestroy` does not execute without `super.onDestroy()` in `MultiCombobox`